README.md

```diff
@@ -261,11 +261,130 @@ To uncordon a nodegroup, run:
 eksctl drain nodegroup --cluster=<clusterName> --name=<nodegroupName> --undo
 ```
 
+### Cluster Upgrades
+
+An _`eksctl`-managed_ cluster can be upgraded in 3 easy steps:
+
+1. update control plane version with `eksctl update cluster`
+2. replace each of the nodegroups by creating a new one and deleting the old one
+3. update default add-ons:
+  - `kube-system:deployment/kube-dns` or `kube-system:deployment/coredns`
+  - `kube-system:ds/kube-proxy`
+  - `kube-system:ds/aws-node`
+
+Please make sure to read this section in full before you proceed.
+
+> NOTE: Kubernetes supports version drift of up-to 2 minor versions during upgrade process.
+
+### Updating control plane version
+
+Control version updates must be done for one minor version at a time.
+
+To update control plane to the next available version run:
+```
+eksctl update cluster --cluster=<clusterName>
+```
+
+This command will not apply any changes right away, you will need to re-run it with
+`--dry-run=false` to apply the changes.
+
+#### Updating nodegroups
+
+You should update nodegroups only after you ran `eksctl update cluster`.
+
+If you have a simple cluster with just an initial nodegroup (i.e. created with
+`eksctl create cluster`), the process is very simple.
+
+Get the name of old nodegroup:
+```
+eksctl get nodegroups --cluster=<clusterName>
+```
+> NOTE: you should see only one nodegroup here, if you see more - read the next section
+
+Create new nodegroup:
+```
+eksctl create nodegroup --cluster=<clusterName>
+```
+
+Delete old nodegroup:
+```
+eksctl delete nodegroup --cluster=<clusterName> --name=<oldNodeGroupName>
+```
+
+##### Updating multiple nodegroups
+
+If you have multiple nodegroups, it's your responsibility to track how each one was configured.
+You can do this by using config files, but if you haven't used it already, you will need to inspect
+your cluster to find out how each nodegroup was configured.
+
+In general terms, you are looking to:
+- review nodegroups you have and which ones can be delete or must be replaced with new version
+- note down configuration of each nodegroup, consider using config file to ease upgrades next time
+
+To create a new nodegroup:
+```
+eksctl create nodegroup --cluster=<clusterName>
+```
+
+To delete old nodegroup:
+```
+eksctl create nodegroup --cluster=<clusterName>
+```
+
+If you are using config file, you will need to do the following.
+
+Get the list of old nodegroups:
+```
+old_nodegroups="$(eksctl get ng --cluster=<clusterName> --output=json | jq -r '.[].Name')"
+```
+
+Edit config file to add new nodegroups.
+
+If you chose to deleted definition of old nodegoups from the config file, then you can run:
+```
+eksctl create nodegroup --config-file=<path>
```

Comment:
I'm not sure I understand this part, will this command delete nodegroups not in the config file?

Reply:
No, it will only create new ones, and unless you've deleted the definitions of the old ones before hand - it will complain about those. You'd want to set `--only=<listOfNewNGs>` otherwise.

Reply:
I think `--only` is documented quite clearly just below this.